README.md

```diff
@@ -3,6 +3,14 @@
 This repository contains code for the Elastic Open Web Crawler.
 This is a tool to allow users to easily ingest content into Elasticsearch from the web.
 
+⚠️ _The Open Crawler is currently in **tech-preview**_.
+Tech-preview features are subject to change and are not covered by the support SLA of generally available (GA) features.
+Elastic plans to promote this feature to GA in a future release.
+
+ℹ️ The Open Crawler requires a running instance of Elasticsearch to index documents into.
+If you don't have this set up yet, check out the [quickstart guide for Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/master/quickstart.html) to get started.
```

Comment:
```suggestion
If you don't have this set up yet, check out the [quickstart guide for Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/master/quickstart.html) to get started.
```

We'll want to suggest Cloud trial first and alternatively quickstart for quick-and-dirty self-managed Docker option without security